draft-trammell-taps-interface.md

```diff
@@ -1001,36 +1003,34 @@ application to receive the Content multiple times.
 0RTT data on Initiate. Probably a transport parameter]
 \[MICHAEL: why? As an app programmer, I can just use Send instead of Initiate.]
 
-#### Checksum {#send-checksum}
+#### Integritycheck_length {#integritycheck_length}
 
 This numeric property specifies the length of the checksum to be used on the Content.
 A value of 0 means that no checksum is required, and a special value (e.g. -1) can
 be used to indicate full checksum coverage (which is also the default). Only
 full coverage is guaranteed, any other requests are advisory.
 
-###Protocol Properties
+#### ACK_immediately {#ack_immediately}
+
+This boolean property specifies, if true, that an application wants this
+Content to be immediately acknowledged by the receiver. In case of
+reliable transmission, this informs the transport protocol on the sender
+side faster that it can remove the Content from its buffer; therefore
+this property can be useful for latency-critical applications that maintain
+tight control over the send buffer (see {{#sending}}).
 
```

Comment:
I like this property, but I left it out because I wasn't sure I knew of any protocols that supported it...

Reply:
SCTP does. RFC 7053. It's really quite a special thing... I think you have to be quite an advanced application writer doing advanced juggling with the Sent event to make use of this (Sent will fire when it the transport buffer below drains, which will happen faster when ACKs come in faster). But let them have all the things that can reduce their latency, I'd say.